library/src/cache/updater_state.rs

```diff
@@ -44,6 +44,8 @@ pub struct UpdaterState {
 /// Written out to disk as a json file at STATE_FILE_NAME.
 #[derive(Debug, Deserialize, Serialize)]
 struct SerializedState {
+    /// The client ID for this device.
```

Comment:
What's the expected lifetime of this id?  Can we document that?  It looks like as-implemented its infinite?